spec/index.bs

```diff
@@ -10915,8 +10967,20 @@ When executing encoded render pass commands as part of a {{GPUCommandBuffer}}, a
 
     : <dfn>\[[stencilReference]]</dfn>, of type {{GPUStencilValue}}
     ::  Current stencil reference value, initially `0`.
+
+    : <dfn>\[[colorAttachments]]</dfn>, of type [=sequence=]&lt;{{GPURenderPassColorAttachment}}?&gt;
+    ::  The color attachments and state for this render pass.
+
+    : <dfn>\[[depthStencilAttachment]]</dfn>, of type {{GPURenderPassDepthStencilAttachment}}?
+    ::  The depth/stencil attachment and state for this render pass.
 </dl>
 
+Render passes also have <dfn dfn>framebuffer memory</dfn>, which contains the [=texel block|texel=] data associated with
+each attachment that is written into by draw commands and read from for blending and depth/stencil testing.
```

Comment:
I went back and forth a bit on whether it's clearer to describe "framebuffer memory" as a thing that's separate from the attachment texture's own memory. On the one hand it's unnecessary complexity (we could describe everything as if non-tiled), on the other hand the "load"/"store"/"discard" names only make sense if it's separate. But the complexity probably doesn't actually change because if we don't describe it normatively, we have to describe it non-normatively (in order to justify the names and performance characteristics).

So I think describing it this way makes sense.

Reply:
Sounds like we went through the same mental process and arrived at the same place. :smile: It's awkward no matter how we frame it, I think.